doc/plutus/troubleshooting.rst

```diff
@@ -97,23 +97,24 @@ To reduce code size, on-chain errors only output codes. Here's what they mean:
 
 - Prelude errors
 
-  - ``P0: PlutusTx.Enum.().succ: bad argument``
-  - ``P1: PlutusTx.Enum.().pred: bad argument``
-  - ``P2: PlutusTx.Enum.().toEnum: bad argument``
-  - ``P3: PlutusTx.Enum.Bool.succ: bad argument``
-  - ``P4: PlutusTx.Enum.Bool.pred: bad argument``
-  - ``P5: PlutusTx.Enum.Bool.toEnum: bad argument``
-  - ``P6: PlutusTx.Enum.Ordering.succ: bad argument``
-  - ``P7: PlutusTx.Enum.Ordering.pred: bad argument``
-  - ``P8: PlutusTx.Enum.Ordering.toEnum: bad argument``
-  - ``P9: PlutusTx.List.!!: negative index``
-  - ``Pa: PlutusTx.List.!!: index too large``
-  - ``Pb: PlutusTx.List.head: empty list``
-  - ``Pc: PlutusTx.List.tail: empty list``
-  - ``Pd: Check has failed``
```

Comment:
E.g.
```
Command failed: transaction build  Error: The following scripts have execution failures:
the script for policyId 0 (in the order of the PolicyIds) failed with: 
The Plutus script evaluation failed: An error has occurred:  User error:
The provided Plutus code called 'error'.
Script debugging logs: not signed by redeemer pubkeyhash
Pd
```

Reply:
Not in the current master. Actually it was changed two months ago https://github.com/input-output-hk/plutus/commit/75aa0d6158aef2bac834a714029e12acf60b5dae#diff-6960965185719fd9e43176f8e9dccd3c951fd99dbcf3dda72344528582bd3200L142-L143

I suppose "Pd" happens because cardano-node uses an old version of plutus.